src/test/java/com/epam/tc/hw7/entities/UserEntity.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw7.entities;
+
+import com.epam.tc.hw7.PropertiesExtractor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UserEntity {
+
+    public static final UserEntity ROMAN_IOVLEV = new UserEntity(
+            PropertiesExtractor.getUsername(),
+            PropertiesExtractor.getPassword(),
+            PropertiesExtractor.getCorrectUsername());
```

Comment:
Please move constant User to other class with constants
DTO should stay only DTO and should not store it's instances.

Reply:
moved constant User to HomePage